### ISSUE TITLE
Android Build Tools

### ISSUE THREAD

**Issue by divanvb** [beevelop/docker-ionic#118]
Hey man.

First off, thanks for a great docker.

I was wondering if you can help me, I'm trying to build an apk using your docker but I'm running into this issue. Any help would be appreciated.

And do you have a good .yml file for building a production apk with everything minified & aot rendering? Basically, the apk being as fast & as small as possible 😄 

```A problem occurred evaluating root project 'android'. > No installed build tools found. Install the Android build tools version 19.1.0 or higher.```

**Comment by beevelop** [comment]
@divanvb The upstream image (beevelop/android) did not properly install the Android build tools. https://github.com/beevelop/docker-android/pull/19 deals with this issue and should fix the problem. I'll keep this open until I can successfully verify the changes.

**Comment by divanvb** [comment]
@beevelop I've just tried to rerun my pipe and it looks like the build tools error I previously got is fixed, but there is now another issue. It's not a major issue I believe. See below:

```* What went wrong:
A problem occurred configuring root project 'android'.
> You have not accepted the license agreements of the following SDK components:
  [Android SDK Platform-Tools].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html```

**Comment by beevelop** [comment]
Should be fixed with version `v3.16.0`. 👍
